src/scripts/create_release.sh

```diff
@@ -47,7 +47,7 @@ function create_release() {
         --show-error \
         --header "Authorization: apiToken ${!INSTANA_API_TOKEN_NAME}" \
         --header "Content-Type: application/json" \
-        --header "User-Agent: Instana/CircleCI" \
+        --header "User-Agent: instana/pipeline-feedback-orb/1.1" \
```

Comment:
Hard-coding it all but guarantees that it goes out of synch with the very next release. I advise you look into ways of doing this automatically ;-)

Reply:
That was an intermediate step :-) It should respect the ORB_VERSION env variable and use a fixed version as fallback.

Reply:
Who is providing the `ORB_VERSION` env var? Is it built into the CircleCI runtime? Do you have a link to that docu? Just to be pedantic, there is zero value in having the user set the version, it must be compiled in the script. For concourse, we could do it over the Docker image, but here we cannot because out Docker image, the executor, is optional :-(